CHANGES

```diff
@@ -83,6 +83,7 @@ Bugs fixed
 * #11192: Restore correct parallel search index building.
   Patch by Jeremy Maitin-Shepard
 * Use the new Transifex ``tx`` client
+* #9778: Disable localisation of the ``objects.inv`` file during Reproducible Builds
```

Comment:
```suggestion
* #9778: Disable localisation when the ``SOURCE_DATE_EPOCH`` environment
  variable is set, to assist with 'reproducible builds'. Patch by James Addison
```

Reply:
@jayaddison I didn't know if you'd like to be credited as Jay (per your username) or James (also per your username!).

Reply:
Good question - I'm rarely sure myself.  Mostly in commit messages I go by James, so that probably makes sense here.  Thanks for checking :)